packages/salesforcedx-vscode-core/package.json

```diff
@@ -710,11 +710,6 @@
           "type": "boolean",
           "default": false,
           "description": "%push_or_deploy_on_save_enabled_description%"
-        },
```

Comment:
Customers are enabling this setting by mistake so I'm removing it from the Settings menu.

Reply:
How is it enabled now?

Reply:
Like any of the other settings, via the settings.json file or the Settings UI.

Reply:
Gotcha, that's what I figured. Wasn't sure if it was "deregistered" or something by doing this.